source/reference/method/MongoDBClientBulkWrite-updateOne.txt

```diff
@@ -0,0 +1,98 @@
+=====================================
+MongoDB\\ClientBulkWrite::updateOne()
+=====================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 2.1
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\ClientBulkWrite::updateOne()
+
+   Specify an update operation in the bulk write command for the first
+   matching document. This method returns the
+   :phpclass:`MongoDB\ClientBulkWrite` instance on which its called.
+
+   .. code-block:: php
+
+      function updateOne(
+          array|object $filter,
+          array|object $update,
+          array $options = []
+      ): self
+
+Parameters
+----------
+
+``$filter`` : array|object
+  The filter criteria that specifies the documents to update.
+
+``$update`` : array|object
+  Specifies the field and value combinations to update and any relevant update
+  operators. ``$update`` uses MongoDB's :manual:`update operators
+  </reference/operator/update>`. Starting with MongoDB 4.2, an `aggregation
```

Comment:
This note can be removed as the client bulk write feature already requires MongoDB 8.0.

Note that compatibility with MongoDB < 4.2 will be removed in the next driver version (probably 2.3).

Reply:
Sure, I'll remove this note in any place it appears in the bulkwrite-related docs